src/internet_identity/src/main.rs

```diff
@@ -905,6 +905,21 @@ mod openid_api {
     }
 }
 
+/// API required for the discoverable credentials flow
+mod discoverable_credentials {
+    use crate::anchor_management::lookup_anchor_number_and_pubkey_with_credential_id;
+    use crate::DiscoverableCredentialData;
+    use ic_cdk::query;
+    use internet_identity_interface::internet_identity::types::CredentialId;
+
+    #[query]
+    fn get_discoverable_credential_data(
```

Comment:
This was called `get_pubkey_by_credential_id` in the candid interface: https://github.com/dfinity/internet-identity/compare/main...to-be-shipped-hackathon#diff-623401f1d9297ecff25efc7dc8e6e87983a4d8e0b62e82b8e029375805708392R815